app/controllers/movies_controller.rb

```diff
@@ -1,7 +1,8 @@
 class MoviesController < ApplicationController
-    PER_PAGE = 10
+  PER_PAGE = 10
```

Comment:
最後列が中途半端ですので，1,2,3の公倍数である6の倍数にしておきましょう。

```suggestion
  PER_PAGE = 12
```